classes/class-bsf-custom-fonts-render.php

```diff
@@ -218,6 +216,17 @@ public function bb_custom_fonts( $bb_fonts ) {
 			return array_merge( $bb_fonts, $custom_fonts );
 		}
 
+		/**
+		 * Enqueue Block Assets Scripts
+		 *
+		 * @since 1.3.3
+		 */
+		public function add_block_assets_style() {
+			if ( is_admin() && ! is_customize_preview() ) {
```

Comment:
@shubhw12 Custom fonts are already loaded through this action- [code](https://github.com/brainstormforce/custom-fonts/pull/51/files#diff-078096c06e298452c352eb3bed0176fdf8c30577dec1e57410ac166751f0fecfR127)

I was loaded twice which was causing **Headers already sent issue**. That is why we added this condition so that it wont be loaded twice in the customizer.

Reply:
Okay